.gitignore

```diff
@@ -25,3 +25,5 @@ build
 node_modules
 CMakeSettings.json
 xcuserdata/
+/cmake-build-debug/
```

Comment:
This change is useful, but out of scope for this PR

Reply:
Oh, I add it by accident. Sure.